examples/eventloop/c_eventloop.c

```diff
@@ -32,7 +37,7 @@ typedef struct {
 	int removed;      /* timer has been requested for removal */
 
 	/* The callback associated with the timer is held in the "global stash",
-	 * in <stash>.TIMERS_SLOT_NAME[String(id)].  The references must be deleted
+	 * in <stash>.eventTimers[String(id)].  The references must be deleted
```

Comment:
Why is this changed back?

Reply:
Only because it seemed more consistent with the rest of the comments in the code.

Reply:
Ok, just wondering because it was changed to `TIMERS_SLOT_NAME` in the previous pull.

Reply:
Right. Again, only because there are many other comments that reference eventTimers, and only this one was using TIMERS_SLOT_NAME.

Reply:
(which means, my previous change was probably a mistake...)